src/lib.rs

```diff
@@ -589,7 +595,7 @@ impl Default for CertificateParams {
 		// not_before and not_after set to reasonably long dates
 		let not_before = date_time_ymd(1975, 01, 01);
 		let not_after = date_time_ymd(4096, 01, 01);
-		let mut distinguished_name = DistinguishedName::new();
+		let mut distinguished_name = DistinguishedName::default();
```

Comment:
Please don't use Default here, I'm not a fan of it. I like new far more.

Reply:
Why? You are using `Default::default()` at some other places, too. I think the `Default` implementation here should be coherent with concluded calls of `default()`.